src/Context/ContextConfig.php

```diff
@@ -60,7 +61,7 @@ protected function __construct(array $values) {
    * @param string $property_path
    *   A valid property path; e.g., "node.uid.target_id".
    *
-   * @throws \LogicException
+   * @throws \Drupal\rules\Exception\RulesLogicException
```

Comment:
I think we don't need the "Rules" prefix for the exception name. LogicException in the rules namespace is enough.